kolibri/plugins/management/assets/src/views/user-page/user-create-modal.vue

```diff
@@ -119,44 +117,61 @@
       LEARNER: () => UserKinds.LEARNER,
       COACH: () => UserKinds.COACH,
       ADMIN: () => UserKinds.ADMIN,
-      statusMessage() {
-        if (this.errorMessage) {
-          return this.errorMessage;
-        } else if (this.confirmationMessage) {
-          return this.confirmationMessage;
+      usernameAlreadyExists() {
+        const index = this.users.findIndex(user => user.username === this.username);
+        if (index === -1) {
+          return false;
         }
-        return false;
+        return true;
+      },
+      usernameNotAlphaNum() {
+        if (this.username === '') {
+          return false;
+        }
+        const re = /^\w+$/g;
+        return !re.test(this.username);
```

Comment:
I put that check in because I don't want the textbox to appear invalid if it is empty.